src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMappingTest.java

```diff
@@ -105,6 +107,7 @@ class HearingsDetailsMappingTest extends HearingsMappingBase {
     @BeforeEach
     void setUp() {
         caseData = SscsCaseData.builder()
+            .adjournment(Adjournment.builder().adjournmentInProgress(YesNo.NO).build())
```

Comment:
Is this required? If the feature flag is disabled, adjournment in progress is irrelevant.